custom_tesseract_training/README.md

```diff
@@ -1,78 +1,293 @@
-  These two commands should fix it. run `sudo apt-get update` and then run `sudo apt-get install ffmpeg libsm6 libxext6 -y`
-  Also note that you will have an error if the image path and custom model names (`lang = 'storysquad'` are incorrect
-- There is an updated `ocr_inference.py` script, `models_comparison.py` found in the `models_comparison.py` which will
-  let you compare multiple images and models including google vision OCR. check the `model_comparison` dir for
-  instructions on how to run it in your IDE.
-  
-##what is next?
-    - create more training data
-    - build a preprocessing pipeline
-    - hyperparameter tuning
-    - automate the OCR_performance record process and connect the it an database
+#Workflow for Fine Tuning Tesseract Model in Docker 
+##A guide for using Docker to fine tune Story Squad’s Tesseract model
+
+###Contents:
+* Connecting to Docker
+* Data Storage
+* Selecting the Hyperparameters
+* Fine Tuning the Model
+* Saving the Model
+* Model Performance Logging
+* Testing Model Against Baseline
+* Previous Readme
+
+###Prerequisites: 
+* Make sure you’ve pulled the latest main branch
+* Make sure the data has already been preprocessed, in the agreed upon conventions
+
+##Connecting to Docker
+
+(must have the DS repo cloned (https://github.com/Lambda-School-Labs/scribble-stadium-ds))
+
+(all commands done in the terminal, with Docker open, in your base environment (you will connect to a virtual environment in docker)
+
+* Run the Dockerfile using this command from the repo root. 
+  * docker-compose -f docker.dev.yml up -d
+    *  Note: You only need to do this step once.
+   
+* Run this command to access the container once the build is complete, or if you have already built the container before
+  * docker exec -ti scribble-ocr bash
+* cd to root
+  * cd ..
+* cd to train folder
+  * cd train
+* cd to tesstrain folder
+  * cd tesstrain
+* activate the virtual environment
+  * source ocr/bin/activate
+* install the required libraries:
+  * pip install -r requirements.txt
+
+####For further information on the Dockerfile, see here: ​​https://github.com/BloomTech-Labs/scribble-stadium-ds/blob/main/Dockerfile_tesseract_training
+
+##Data Storage
+When using Docker all new training data must be placed in:
+
+/train/tesstrain/data/storysquad-ground-truth
+###Training Data and Label Naming Conventions
+* Preprocess Data using agreed upon pipeline
+* Label the data by transcribing the characters exactly how they appear (not fixing the spelling) in a plain .gt.txt file
+* Ensure that images are saved with the .tif or .png extension and that the exact transcription (ground truth labels) are saved in .gt.txt
+* Verify that name of each training example/label pair are exactly identical (excluding the .gt.txt and .tif extensions)
+###Adding New Training Data in Docker
+* Open the docker container using these instructions:
+https://docs.google.com/document/d/16NF5IxDxllME934f9YaepYScPrShJJt10i5gEzErzgQ/edit#
```

Comment:
Would it be too much to include these instructions here? I really like seeing the terminal commands step by step. If we could see that for this as well, it might help with the work flow.

Reply:
Great suggestion - I've made a note and will include it in the next patch (when we add instructions for preprocessing or storing the data in the cloud). Thanks Carl